### PR TITLE
feat: Added acct, region, api, resource commands and improved ux for deploy

### PR DESCRIPTION
**Pull request by Aerex**
# Context

This PR will add a few new VSCode commands as well as improve on UX for deploy. The following commands will be added:

- `ibmcloud account list`
- `ibmcloud account show`
- `ibmcloud account users`
- `ibmcloud api`
- `ibmcloud regions`
- `ibmcloud target`
- `ibmcloud resource service-instances`

In addition, the README has been updated to removed the old YouTube links and clarified some of the dev flow instructions.

# Breaking Change
- The `ibmcloud dev deploy` command will only allow deploying to IBM Cloud Kubernetes Service

# Steps To Test

## Setup
1. Install `vsce` npm package if not already: `npm install -g vsce`
2. Checkout PR branch
3. Run the command `vsce package`
4. Verify vsix package is built successfully
5. Open a starterkit project in VSCode
6. Open the `.ibm-project` file and remove the values for the `image-deploy-target` and `cluster-name` if values exist
    - The `dev deploy` command will use the values in these properties to determine what cluster and image target to deploy. If they are not valid the deployment will fail
7. Go to **Extension -> Install From VSIX`** and select vsix file created in step 4

8. Reload VSCode when requested

## Test new commands
**Note:** 

1. Make sure to follow the steps in the **Setup** section
2. Open the command palette 
  - To enter Command Palette press `Ctrl+Shift+P` or `F1` or navigate to `View-> Command Palette`
3. Enter `ibmcloud login --sso`
4. Follow the instructions to login into IBM Cloud
5. Repeat step 2 (command palette) 
6. Enter the command `ibmcloud account list`
8. Verify that step 6 runs successfully
9. Repeat steps 5-7 except use the following commands: 
    - `ibmcloud account show`
    - `ibmcloud account users`
    - `ibmcloud api`
    - `ibmcloud regions`
    - `ibmcloud target`
    - `ibmcloud resource service-instances`

## Test new UX for dev deploy
**Note:** Before proceeding make sure that you have a working IBM Kubernetes Service (IKS) cluster and an image namespace in IBM Container Registry (CR)

1. Make sure to follow the steps in the **Setup** section
2. Login into to IBM Cloud by entering the command `ibmcloud login --sso` in the Command Palette
3. Verify login is successful
4. Build the project by entering the command `ibmcloud dev build (release)` in the Command Palette
5. Verify build is successful
6. Start deployment by entering the command `ibmcloud dev deploy` in the Command Palette
7. Verify that terminal window is opened with the `ibmcloud dev deploy` executed
8. Follow the prompts provided and verify that app is deployed successfully.
13. Click on the link at the bottom of the terminal output and verify that app can be viewed in the browser

# Videos
new_dev_deploy_iks

### CODE REVIEW

**Comment by Aerex** [comment]
@steveclay @schmidtp0740 Updated `dev deploy` command to utilize terminal to handle deployment.

File: src/util/DeployCommand.ts [hunk 1]
```diff
@@ -1,93 +0,0 @@
-/**
```
Comment (Aerex) [inline src/util/DeployCommand.ts:1]: **Callout**: Since we are using the terminal we don't need this file anymore.

File: .travis.yml [hunk 2]
```diff
@@ -5,7 +5,8 @@ services:
 
 language: node_js
 
-node_js: node
+node_js: 
+  - 16
```
Comment (Aerex) [inline .travis.yml:9]: **Callout**: Currently there is an issue with the latest version (node18)  requiring  dependencies that are not built with travis build machine. Dropping to v16 for compatibility reasons.

File: tsconfig.json [hunk 3]
```diff
@@ -1,6 +1,7 @@
 {
     "compilerOptions": {
         "module": "commonjs",
+        "resolveJsonModule": true,
```
Comment (Aerex) [inline tsconfig.json:4]: **Callout:** Added to allow importing json files used in sourcecode. See [extensions.ts#L24-L25](https://github.com/IBM-Cloud/ibm-developer-extension-vscode/blob/feat/add-new-dev-cli-cmds/src/extension.ts#L24-L25) for usecase

**Comment by steveclay** [comment]
Probably should indicate in the README what the breaking change was --just one short line, not very verbose. I don't think I saw that there.

**Comment by Aerex** [comment]
@steveclay Added tidbit on breaking change in README

File: README.md [hunk 4]
```diff
@@ -43,42 +54,30 @@ Easily invoke commands from the IBM Cloud CLI from directly inside of the VS Cod
...
     - [IBM Cloud CLI Documentation](https://cloud.ibm.com/docs/apps?topic=apps-create-deploy-app-cli)
-- Open the *project’s folder* locally in the VS Code editor
+- Open the *project’s folder* in VS Code
+    - Press Ctrl-K+Ctrl-O or navigate to File -> Open Folder to select folder
 - Use the `ibmcloud dev build` command to build the app into a Docker image
 - Use the `ibmcloud dev debug` command to run the app in local Docker for development
 - Use the `ibmcloud dev run` command to run the app in local Docker in release mode
 - Use the `ibmcloud dev deploy` command to deploy the app (in the Docker container) to IBM Cloud
 - Use the `ibmcloud dev view` command to open your deployed project on IBM Cloud in a web browser
-- Use the `ibmcloud dev console` command to open the IBM Cloud console for your project in a web browser
```
Comment (steveclay) [inline README.md:59]: I don't recall the discussion around removing `dev console`. Can you remind me why this was done?

**Review by steveclay — DISMISSED** [review]
One question. Otherwise looks good.

File: README.md (hunk 4 above)
Comment (Aerex) [reply README.md:59]: Sorry. I think that was a mistake. The code is still there but I must have accidentally remove this line from the README. I have readded. Thanks for catching that.

**Review by steveclay — APPROVED** [review]

**Pull request merged**